### DISCUSSION TITLE
请教paddlespeech转换音频得到文字的同时，怎么获得每个句子对应的开始时间和结束时间，就像字幕文件那样？

### DISCUSSION THREAD

**Discussion by richard201410** [PaddlePaddle/PaddleSpeech#1267] [Q&A] [answered]
请教paddlespeech转换音频得到文字的同时，怎么获得每个句子对应的开始时间和结束时间，就像字幕文件那样？

字幕文件样例：
00:03:23,760 --> 00:03:25,860 如何将直线拟合到数据。

**Comment by zh794390558** [comment, marked as answer]
目前不支持这个功能。需要自己加上vad才可以，可以试试webrtc vad。

**Reply by richard201410** [reply]
@zh794390558 谢谢。已经解决。
